File: Persistencia_Cpp/src/Student.cpp
#include "../include/Student.hpp"

// Constructor
Student::Student()
{

}

// Setters
void Student::setIdentifier(int identifier)
{
    m_identifier = identifier;
}

void Student::setName(string name)
{
    m_name = name;
}

void Student::setSurname(string surname)
{
    m_surname = surname;
}

// Getters
int Student::getIdentifier()
{
    return m_identifier;
}

string Student::getName()
{
    return m_name;
}

string Student::getSurname()
{
    return m_surname;
}

// Destructor
Student::~Student()
{

}

File: Persintencia_Python/include/Student.py
class Student:
    def __init__(self):
        self.m_identifier = 0
        self.m_name = ""
        self.m_surname = ""
        
    # Setters
    def setIdentifier(self, identifier: int):
        self.m_identifier = identifier
    
    def setName(self, name: str):
        self.m_name = name
    
    def setSurname(self, surname: str):
        self.m_surname = surname
    
    # Getters
    def getIdentifier(self) -> int:
        return self.m_identifier
    
    def getName(self) -> str:
        return self.m_name
    
    def getSurname(self) -> str:
        return self.m_surname
    
    def __del__(self):
        pass